packages/app-store/_components/AppCard.tsx

```diff
@@ -34,6 +35,7 @@ export default function AppCard({
   disableSwitch?: boolean;
   switchTooltip?: string;
   hideSettingsIcon?: boolean;
+  eventType: EventTypeAppSettingsComponentProps["eventType"];
```

Comment:
See my comment on `OmniInstallAppButton` we should only pass down the event type id.